docs/DeveloperGuide.md

```diff
@@ -257,27 +257,46 @@ _{Explain here how the data archiving feature will be implemented}_
 
 **Target user profile**:
 
-* has a need to manage a significant number of contacts
+* home-based small business owners who are managing orders and inventory information efficiently
```

Comment:
I think `who want to manage their orders and inventory more efficiently` would be better here

Reply:
Otherwise, to follow our original target user profile:
`Home-based small business owners who are managing orders and customer information inefficiently`